src/main/java/newbank/server/microloans/LoanRequest.java

```diff
@@ -0,0 +1,29 @@
+package newbank.server.microloans;
+
+import newbank.server.Customer;
+import org.javamoney.moneta.Money;
+
+import static newbank.Configuration.MICROLOAN_REQUEST_EXPIRY_DAYS;
+
+public class LoanRequest extends LoanProposal {
+    private final Customer borrower;
+
+    public LoanRequest(
+        Money proposedAmount, 
+        Customer borrower, 
+        int repaymentPeriod
+    ) {
+        super(proposedAmount, repaymentPeriod);
+        this.expiryDate = creationDate.plusDays(MICROLOAN_REQUEST_EXPIRY_DAYS);
+        this.borrower = borrower;
+    }
+
+    public Customer getBorrower() {
+        return borrower;
+    }
+
+    @Override
+    public int getRepaymentPeriod() {
```

Comment:
an abstract method or an interface?

Reply:
Ah, great spot! 
Not sure how that happened. I think Intellij was being too helpful at some point :laughing: